File: Programacion multimedia y dispositivos moviles2T/GestorDeTareas/GestorDeTareas/app/src/main/java/com/example/gestordetareas/MainActivity.java
package com.example.gestordetareas;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

import com.example.gestordetareas.vista.DashboardActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Redirigir al DashboardActivity directamente
        Intent intent = new Intent(this, DashboardActivity.class);
        startActivity(intent);
        finish(); // Cierra MainActivity para evitar que quede en segundo plano
    }
}

File: Programacion multimedia y dispositivos moviles2T/ServicioMusicaApp/ServicioMusicaApp/app/src/main/java/com/example/serviciomusicaapp/MainActivity.kt
package com.example.serviciomusicaapp



import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnReproducirMusica = findViewById<Button>(R.id.btnReproducirMusica)
        val btnDetenerMusica = findViewById<Button>(R.id.btnDetenerMusica)

        // Iniciar el servicio al pulsar "Reproducir Música"
        btnReproducirMusica.setOnClickListener {
            val intent = Intent(this, ServicioMusica::class.java)
            startService(intent)
        }

        // Detener el servicio al pulsar "Detener Música"
        btnDetenerMusica.setOnClickListener {
            val intent = Intent(this, ServicioMusica::class.java)
            stopService(intent)
        }
    }
}
